app/controllers/api/v1/good_deeds_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::GoodDeedsController < ApplicationController
```

Comment:
Good thinking on added this additional controller for the home page good deeds. Since this index page is also the home page, this works!